<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverWidget</class>
 <widget class="QWidget" name="SolverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QtCharts::QChartView" name="chartsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>641</width>
     <height>491</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>34</y>
     <width>261</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_matrixItems">
    <item>
     <widget class="QLabel" name="label_paymentMatrix">
      <property name="text">
       <string>Матриця втрат</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table_matrixItems">
      <property name="rowCount">
       <number>4</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>l_i_1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>l_i_2</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_buttons_inc_dec_matrixItems">
      <item>
       <widget class="QPushButton" name="button_decreaseMatrixItems">
        <property name="text">
         <string>Видалити останній рядок</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_increaseMatrixItems">
        <property name="text">
         <string>Додати новий рядок</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_solutionLog">
      <property name="text">
       <string>Інформація про розв'язок</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="text_solutionLog">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_doSolve">
      <property name="text">
       <string>Розв'язати</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_buttons_redirect">
      <item>
       <widget class="QPushButton" name="button_backToPractice">
        <property name="text">
         <string>До практики</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_toTheory">
        <property name="text">
         <string>Теорія</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_toPractice">
        <property name="text">
         <string>Практика</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>911</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Модуль &quot;автоматичного ровз'язку&quot;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
